contracts/modules/common/BaseModule.sol

```diff
@@ -83,4 +83,26 @@ contract BaseModule is Module {
     function isOwner(BaseWallet _wallet, address _addr) internal view returns (bool) {
         return _wallet.owner() == _addr;
     }
+
+    /**
+     * @dev Helper method to invoke a wallet.
+     * @param _wallet The target wallet.
+     * @param _to The target address for the transaction.
+     * @param _value The value of the transaction.
+     * @param _data The data of the transaction.
+     */
+    function invokeWallet(address _wallet, address _to, uint256 _value, bytes memory _data) internal returns (bytes memory _res) {
+        bool success;
+        // solium-disable-next-line security/no-call-value
+        (success, _res) = _wallet.call.value(0)(
+            abi.encodeWithSignature("invoke(address,uint256,bytes)", _to, _value, _data)
```

Comment:
Maybe it is worth defining
`bytes4 constant internal INVOKE = bytes4(keccak256("invoke(address,uint256,bytes)"));`
and use `abi.encodeWithSelector(INVOKE, _to, _value, _data)` to save some gas.

Reply:
Why do you think it should save some gas? In my tests, this makes both the module deployment cost and the method execution cost slightly higher.

Reply:
Really? That's surprising. I would expect that having the signature computed at compile time and stored would be cheaper than computing it at run time, but I guess this statement is making a lot of assumptions about how the compiler works.